stack.yaml

```diff
@@ -46,7 +46,7 @@ extra-deps:
 - hiedb-0.4.3.0
 - implicit-hie-0.1.2.7
 - implicit-hie-cradle-0.5.0.1
-- fourmolu-0.12.0.0
+- fourmolu-0.13.1.0
```

Comment:
I don't think this is necessary anymore. I removed this line in #3764, and the stack snapshot should just be using whatever version of fourmolu is in the snapshot.

And this snapshot is only used for build; cabal is used to build the release artifacts. I think the PR can just be closed